README.md

```diff
@@ -33,284 +29,16 @@ Add the below plugin:
-  
-  val pause = new PauseBuilder(1 milliseconds, None)
-  
-  setup("login", "Login") withActions(navigateToLoginPage, pause, submitLogin)
-
-  runSimulation()
-}
-```
-
-#### Using Gatling's Session API 
-
-Gatling's [Session API](https://gatling.io/docs/3.4/session/session_api/) is used to update
-Gatling's Session. To use the Session API with performance-test-runner, the `ChainBuilder` returned when executing a Session
-API should be converted into an `ActionBuilder`. The `ActionBuilder` then can be chained to a `setup` using `withActions`. 
-
-An example:
-
-```scala
-import io.gatling.core.action.builder.ActionBuilder
-import io.gatling.core.session.Session
-import io.gatling.core.Predef._
-
-/** Executes the Session API to set testId with value perf-12345 in Gatling's Session.
- *  Converts the resulting ChainBuilder to a List[ActionBuilder] 
- */
-val setRandomTestId: List[ActionBuilder] = {
-    exec((session: Session) => session.set("testId", "perf-12345}"))
-  }.actionBuilders
-
-// Chains the setRandomTestId using `withActions`
-setup("prep", "Prepare for test") withActions (setRandomTestId:_*)
-```
-
-### More about the journey configuration.
-
-`description` will be assigned to the journey in the test report
-
-`load` is the number of journeys that will be started every second
-
-`feeder` is the relative path to the csv feeder file. More [here](http://gatling.io/docs/2.1.7/session/feeder.html#csv-feeders)
-
-`parts` is the list of parts that combined create your journey
-
-`run-if` is an optional list of labels. Runs this journey only if a label from this list is passed in the `labels` parameter of application.conf
-
-`skip-if` is an optional list of labels. Skips this journey if a label from this list is passed in the `labels` parameter of application.conf
-
-You can have as many journeys as you like in journeys.conf, the simulation will start and run them all together.
-
-
-### More about the services configuration.
-
-Contains the name of the service and the port when running locally. Read the services-local.conf file for more details and examples.
-
-
-### More about application.conf
-
-`runLocal` boolean value to run test locally. Default value is true.
-
-`baseUrl` is the default url for every service. Read the application.conf file for more details and examples.
-
-`rampupTime` is the time in minutes for the simulation to inject the users with a linear ramp up
-
-`constantRateTime` is the time in minutes for the simulation to inject users at a constant rate
-
-`rampdownTime` is the time in minutes for the simulation to inject the users with a linear ramp down
-
-`loadPercentage` is the percentage of the load for the journeys. Read the application.conf file for more details and examples.
-
-`journeysToRun` contains the journeys that will be executed. Leave it empty if you want to run all the journeys
+Refer to the [getting-started](getting-started.md) guide for implementing your first simulation.
```

Comment:
This link isn't working, it's expecting the referenced readme name to be uppercase how you've named the new file `GETTING-STARTED.md`